admin_guide/out_of_resource_handling.adoc

```diff
@@ -75,243 +165,353 @@ curl <certificate details> \
 
-If an eviction threshold is met, independent of its associated grace period, the
-node reports a condition indicating that the node is under pressure.
+You can control how much of a node resource is made available for scheduling in order to allow the scheduler to fully allocate a node and to prevent
+evictions.
 
-The following node conditions are defined that correspond to the specified
-eviction signal.
+Set `system-reserved` equal to the amount of resource you want available to the scheduler for deploying pods and for system-daemons.
+Evictions should only occur if pods use more than their requested amount of an allocatable resource.
 
-.Node Conditions Related to Low Resources
-[cols="2a,2a,8a",options="header"]
-|===
+A node reports two values:
 
-|Node Condition |Eviction Signal |Description
+* `Capacity`: How much resource is on the machine
+* `Allocatable`: How much resource is made available for scheduling.
 
-|`*MemoryPressure*`
-|`*memory.available*`
-|Available memory on the node has satisfied an eviction threshold.
 
-|`*DiskPressure*`
-|`*nodefs.available*`, `*nodefs.inodesFree*`, `*imagefs.available*`, or `*imagefs.inodesFree*`
-|Available disk space and inodes on either the node’s root file system or image file system has satisfied an eviction threshold.
-|===
+To configure the amount of allocatable resources: 
+
+. Edit the node configuration file (the *_/etc/origin/node/node-config.yaml_* file).
+
+. Add or modify the `system-reserved` parameter for `eviction-hard` or `eviction-soft`
++
+====
+----
+kubeletArguments:
+  eviction-hard: <1>
+    - "memory.available<500Mi"
+  system-reserved:
+    - "1.5Gi"
+----
+<1> This threshold can either be `eviction-hard` or `eviction-soft`.
+====
+
+. Restart the {product-title} services for the changes to take effect:
++
+ifdef::openshift-enterprise[]
+----
+# systemctl restart atomic-openshift-node
+----
+endif::[]
+ifdef::openshift-origin[]
+----
+# systemctl restart origin-node
+----
+endif::[]
+
+
+
+[[out-of-resource-oscillation-of-node-conditions]]
+== Controlling Node Condition Oscillation 
+
+If a node is oscillating above and below a soft eviction threshold, but not exceeding its associated grace period, the corresponding node condition
+oscillates between *true* and *false*, which can cause problems for the scheduler.
+
+To prevent this oscillation, set the `eviction-pressure-transition-period` parameter to control how long the node must wait before transitioning out of a pressure condition.
+
+. Edit or add the parameter to the `kubeletArguments` section of the node configuration file 
+(the *_/etc/origin/node/node-config.yaml_*) 
+using a set of `<resource_type>=<resource_quantity>` pairs.
+----
+kubeletArguments:
+  eviction-pressure-transition-period="5m"
+----
++
+The node will toggle the condition back to *false* when the node has not observed an eviction threshold being met
+for the specified pressure condition for the specified period.
```

Comment:
Mainly because of the "will", I'd suggest:

"The node toggles back to *false* when it has not observed an eviction threshold being met for the specified pressure condition for the specified period."